[2025-04-16T18:05:49.418+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-16T18:05:49.450+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Medium_to_Pinecone.create_pinecone_index scheduled__2025-04-03T00:00:00+00:00 [queued]>
[2025-04-16T18:05:49.464+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Medium_to_Pinecone.create_pinecone_index scheduled__2025-04-03T00:00:00+00:00 [queued]>
[2025-04-16T18:05:49.465+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2025-04-16T18:05:49.497+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): create_pinecone_index> on 2025-04-03 00:00:00+00:00
[2025-04-16T18:05:49.558+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'Medium_to_Pinecone', 'create_pinecone_index', 'scheduled__2025-04-03T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/HW8_Pinecone.py', '--cfg-path', '/tmp/tmpwqg2_e31']
[2025-04-16T18:05:49.573+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=544) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-16T18:05:49.583+0000] {standard_task_runner.py:105} INFO - Job 10: Subtask create_pinecone_index
[2025-04-16T18:05:49.583+0000] {standard_task_runner.py:72} INFO - Started process 561 to run task
[2025-04-16T18:05:50.009+0000] {task_command.py:467} INFO - Running <TaskInstance: Medium_to_Pinecone.create_pinecone_index scheduled__2025-04-03T00:00:00+00:00 [running]> on host 5ae9f2aa9b17
[2025-04-16T18:05:50.246+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Medium_to_Pinecone' AIRFLOW_CTX_TASK_ID='create_pinecone_index' AIRFLOW_CTX_EXECUTION_DATE='2025-04-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-03T00:00:00+00:00'
[2025-04-16T18:05:50.255+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-16T18:06:05.489+0000] {logging_mixin.py:190} INFO - Pinecone index 'semantic-search-fast' created successfully
[2025-04-16T18:06:05.492+0000] {python.py:240} INFO - Done. Returned value was: semantic-search-fast
[2025-04-16T18:06:05.549+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-16T18:06:05.550+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=Medium_to_Pinecone, task_id=create_pinecone_index, run_id=scheduled__2025-04-03T00:00:00+00:00, execution_date=20250403T000000, start_date=20250416T180549, end_date=20250416T180605
[2025-04-16T18:06:05.601+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-16T18:06:05.612+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
